opt/Makefile

```diff
@@ -115,10 +118,17 @@ else
 USE_COVERAGE=off
 endif
 
+ifeq ($(PROFILE),1)
+PROFILE=on
+else
+PROFILE=off
+endif
+
```

Comment:
Problems can arise from it. Either we use `override`, or (better) USE_PROFILE=on|off.

Reply:
thank you for the test fast review @rafie . addressed in the latest commit.